### PR TITLE
Disable allowedScrollTypesMask in tvOS

### LINKED ISSUE #1257: tvOS fails to build
## Description

Latest version of gesture handler breaks tvOS builds.

## Steps To Reproduce

1. Install latest version to a tvOS project, run it on tv sim. Fails to compile

### Expected behavior

Compiles

### Actual behavior

Screen Shot 2020-12-07 at 2 18 53 PM

- React: Moot
- React Native: Moot
- React Native Gesture Handler: 1.9.0

### PR DESCRIPTION
**Pull request by csantos1113**
Move TARGET_OS_TV check up to cover enableTrackpadTwoFingerGesture code

Fixes #1257

Credits @UrbanChrisy . I just opened the PR

### CODE REVIEW

**Review by jakub-gonet — APPROVED** [review]
Nice catch, thanks!

**Comment by brittneykernan** [comment]
@jakub-gonet what's the best way to request a package version bump to include this fix in our project?

**Comment by jakub-gonet** [comment]
There are some options:
- you can wait until we release the next version of RNGH :>
- you can use commit hash in yarn / npm to get code directly from this repo
- or you can use [patch-package](https://www.npmjs.com/package/patch-package) to locally patch your RNGH (or using `postinstall` script directly to patch node_modules, it's your call)

I'd suggest using patch-package, it's the most straightforward way to do that.
On a side note, the next release may take some time, because we're changing internals a bit (changing directory structure and rewriting in TS) and we need to test that thoroughly.

**Comment by brittneykernan** [comment]
Thank you greatly @jakub-gonet for the quick reply and information

**Pull request merged**
